lib/dns/records.js

```diff
@@ -64,14 +64,72 @@ const types = {
   TEXT: 13, // TXT
   LOCATION: 14, // LOC
   // new record
-  MAGNET: 15, // TXT
+  MAGNET: 15, // URI
   DS: 16, // DS
   TLS: 17, // TLSA
   SMIME: 18, // SMIMEA
   SSH: 19, // SSHFP
   PGP: 20, // OPENPGPKEY
   // new record
-  ADDR: 21 // TXT
+  ADDR: 21 // URI
+};
+
+/**
+ * Record Types By Value
+ * @constant
+ */
+
+const typesByVal = {
+  [types.INET4]: 'INET4',
+  [types.INET6]: 'INET6',
+  [types.ONION]: 'ONION',
+  [types.ONIONNG]: 'ONIONNG',
+  [types.NAME]: 'NAME',
+  [types.GLUE]: 'GLUE',
+  [types.CANONICAL]: 'CANONICAL',
+  [types.DELEGATE]: 'DELEGATE',
+  [types.NS]: 'NS',
+  [types.SERVICE]: 'SERVICE',
+  [types.URI]: 'URI',
+  [types.EMAIL]: 'EMAIL',
+  [types.TEXT]: 'TEXT',
+  [types.LOCATION]: 'LOCATION',
+  [types.MAGNET]: 'MAGNET',
+  [types.DS]: 'DS',
+  [types.TLS]: 'TLS',
+  [types.SMIME]: 'SMIME',
+  [types.SSH]: 'SSH',
+  [types.PGP]: 'PGP',
+  [types.ADDR]: 'ADDR'
+};
+
+/**
+ * DNS RR Types by Handshake
+ * record type.
+ * @constant
+ */
+
+const rrTypeByType = {
```

Comment:
Nit: `rrTypesByType`

Also open to an alternative name to this if there are any suggestions.

Reply:
`hnsTodns`